src/integration/integration_api_scheduler_test.go

```diff
@@ -70,6 +70,34 @@ var _ = Describe("Integration_Api_Scheduler", func() {
 				Expect(resp.StatusCode).To(Equal(http.StatusNotFound))
 				resp.Body.Close()
 			})
+
+			Context("Scheduler is down", func() {
+				BeforeEach(func() {
+					stopScheduler(schedulerProcess)
+
+					policyStr = readPolicyFromFile("fakePolicyWithSchedule.json")
+					resp, err := attachPolicy(appId, policyStr)
+					Expect(err).NotTo(HaveOccurred())
+					Expect(resp.StatusCode).To(Equal(http.StatusInternalServerError))
+					resp.Body.Close()
+
+					schedulerProcess = startScheduler()
```

Comment:
You don't need to verify the scheduler since the API server will tell us the answer.